### PR TITLE
sqlparser: support for SHOW SCHEMAS

### PR DESCRIPTION
**Pull request by mcuadros**
Signed-off-by: Máximo Cuadros <mcuadros@gmail.com>

This add support for `SHOW SCHEMAS` as alias to `SHOW DATABASES`
https://dev.mysql.com/doc/refman/8.0/en/show-databases.html

### CODE REVIEW

**Comment by kuba--** [comment]
Somehow `shard 1` is failing...

**Comment by kuba--** [comment]
Shard-1 logs:
```
2019/04/04 01:45:02 mysql57.cache_invalidation: running (try 1/3)...
2019/04/04 01:45:21 mysql57.cache_invalidation: WARNING: can't remove temporary VTDATAROOT: remove /tmp/vt_624331230/test-c39/zk_001/version-2/log.1: permission denied

--- FAIL: TestWorfklowGenerator (0.00s)
    workflow_test.go:58: cannot create resharding workflow: there are 2 vtworkers, 1 destination shards: the number should be same
```

**Comment by kuba--** [comment]
Can we restart a build for Shard-1?

**Comment by kuba--** [comment]
@sougou ?

**Comment by sougou** [comment]
Sorry, I'm behind on reviews. I plan to clear the backlog this weekend.

File: go/vt/sqlparser/sql.y [hunk 1]
```diff
@@ -181,7 +181,7 @@ func skipToEnd(yylex interface{}) {
 %token <bytes> NULLX AUTO_INCREMENT APPROXNUM SIGNED UNSIGNED ZEROFILL
 
 // Supported SHOW tokens
-%token <bytes> COLLATION DATABASES TABLES VITESS_KEYSPACES VITESS_SHARDS VITESS_TABLETS VSCHEMA VSCHEMA_TABLES VITESS_TARGET FULL PROCESSLIST COLUMNS FIELDS ENGINES PLUGINS
```
Comment (sougou) [inline go/vt/sqlparser/sql.y]: Could you add `SCHEMAS` to the unreserved keywords list?

File: go/vt/callinfo/plugin_grpc.go [hunk 2]
```diff
@@ -23,27 +23,12 @@ import (
...
-	callinfo := &gRPCCallInfoImpl{
-		method: method,
-	}
-	peer, ok := peer.FromContext(ctx)
-	if ok {
-		callinfo.remoteAddr = peer.Addr.String()
-	}
-
-	return NewContext(ctx, callinfo)
+	return NewContext(ctx, nil)
```
Comment (sougou) [inline go/vt/callinfo/plugin_grpc.go]: I presume this change was unintentional?

Comment (kuba--) [reply go/vt/callinfo/plugin_grpc.go]: @mcuadros - ^^^

**Comment by kuba--** [comment]
Can we re-push or re-run `shard0` tests?

**Pull request merged**